src/darwin/Framework/CHIP/MTRDeviceController+XPC.mm

```diff
@@ -29,6 +29,11 @@
 static NSString * const kMinIntervalKey = @"minInterval";
 static NSString * const kMaxIntervalKey = @"maxInterval";
 
+// Classes that are used by MTRDevice responses.  In particular, needs to
+// include NSError.
+static NSSet * const kXPCAllowedClasses = [NSSet
```

Comment:
Would be good to change this into a static function with a dispatch_once, since static initialiser like this run at framework load time

Reply:
Can do.

Reply:
Actually, going to use a static function with a function-local static NSSet.